WordPress/Classes/Utility/Editor/GutenbergSettings.swift

```diff
@@ -49,18 +83,54 @@ class GutenbergSettings {
     func mustUseGutenberg(for post: AbstractPost) -> Bool {
         if post.isContentEmpty() {
             // It's a new post
-            return isGutenbergEnabled()
+            return isGutenbergEnabled
         } else {
             // It's an existing post
             return post.containsGutenbergBlocks()
         }
     }
+
+    // MARK: - Editor settings sync with remote
+
+    /// Posts the current local editor setting to remote. This local setting will be set to all the user's blogs.
+    ///
+    func setToRemote() {
+        let currentSettings: EditorSettings = isGutenbergEnabled ? .gutenberg : .aztec
+        let allBlogs = getAllBlogs()
+        let delay: TimeInterval = allBlogs.count > 5 ? 0.3 : 0
+
+        queue.async {
+            allBlogs.forEach({ (blog) in
+                self.postEditorSettings(currentSettings, to: blog)
+                Thread.sleep(forTimeInterval: delay)
+            })
+        }
+    }
+
+    private func getAllBlogs() -> [Blog] {
```

Comment:
I think you only want WordPress.com blogs here, so maybe check `defaultAccount.blogs` instead?